gems/pending/appliance_console.rb

```diff
@@ -282,34 +282,57 @@ module ApplianceConsole
         next if new_city == CANCEL
 
         clear_screen
-        say("Date and Time Configuration\n\n")
-
-        new_date = ask_for_date("current date (YYYY-MM-DD)")
-        new_time = ask_for_time("current time in 24 hour format (HH:MM:SS)")
-
-        clear_screen
         say(<<-EOL)
-Date and Time Configuration
+Timezone Configuration
 
         Timezone area: #{tz_area}
         Timezone city: #{new_city}
-        Date:          #{new_date}
-        Time:          #{new_time}
 
           EOL
 
-        if agree("Apply time and timezone configuration? (Y/N): ")
-          say("Applying time and timezone configuration...")
+        if agree("Apply timezone configuration? (Y/N): ")
+          say("Applying timezone configuration...")
           begin
             LinuxAdmin::TimeDate.system_timezone = "#{new_loc}/#{new_city}"
-            LinuxAdmin::TimeDate.system_time = Time.parse("#{new_date} #{new_time}")
           rescue LinuxAdmin::TimeDate::TimeCommandError => e
-            say("Failed to apply time and timezone configuration")
-            Logging.logger.error("Failed to apply time and timezone configuration: #{e.message}")
+            say("Failed to apply timezone configuration")
+            Logging.logger.error("Failed to timezone configuration: #{e.message}")
           end
           press_any_key
         end
 
+      when I18n.t("advanced_settings.datetime")
+        say("Date and Time Configuration\n\n")
+        say("Requires Automatic time synchronization be disabled\n\n")
+
+        if agree("Disable Automatic time synchronization? (Y/N): ")
+          say("Automatic time synchronization will be disabled...")
+
+          new_date = ask_for_date("current date (YYYY-MM-DD)")
+          new_time = ask_for_time("current time in 24 hour format (HH:MM:SS)")
+
+          clear_screen
+          say(<<-EOL)
+Date and Time Configuration
+
+          Date:          #{new_date}
+          Time:          #{new_time}
+
+            EOL
+
+          if agree("Apply time configuration? (Y/N): ")
+            say("Applying time configuration...")
+            begin
+              LinuxAdmin::Service.new("chronyd").stop.disable
```

Comment:
@carbonin Thank you!

I had considered doing that but from the users perspective they are changing the date/time. Disabling chrony is an implementation detail so I combined them. I totally see it could go ether way. So I will do as you suggest.